tests/websocket/test_streamer.py

```diff
@@ -39,7 +39,11 @@ def test_streamer(self) -> None:
             addresses.append(AddressItem(idx, wallet.get_address(wallet.get_key_at_index(idx))))
 
         # Create the expected result.
-        expected_result: list[dict[str, Any]] = [{'type': 'stream:history:begin', 'id': stream_id}]
+        expected_result: list[dict[str, Any]] = [{
+            'type': 'stream:history:begin',
+            'id': stream_id,
+            'sliding_window_size': 10000
```

Comment:
Currently the `window_size` is configured into the history streamer after the instance is constructed, so if we want this window size of 10000 to be valid we should configure it manually before line 82 (this is the reason for the tests failing I believe).
The alternative would be (as suggested on another comment) to move the window_size to the contructor and also add this 10000 on line 81.

Reply:
There were other outdated things in this test.